lms/static/js/views/fields.js

```diff
@@ -68,14 +94,14 @@
             },
 
             title: function (text) {
-                return this.$('.u-field-title').html(text);
+                return this.$('.u-field-title').html(HtmlUtils.escape(text));
             },
 
             getMessage: function(message_status) {
                 if ((message_status + 'Message') in this) {
                     return this[message_status + 'Message'].call(this);
                 } else if (this.showMessages) {
-                    return this.indicators[message_status] + this.messages[message_status];
+                    return HtmlUtils.HTML(this.indicators[message_status] + this.messages[message_status]);
```

Comment:
It isn't obvious that these are safe to start with.  First alternative would be:
```
return HtmlUtils.HTML(HtmlUtils.escape(this.indicators[message_status]) + HtmlUtils.escape(this.messages[message_status]));
```
Another alternative would be to introduce a new helper like:
```
HtmlUtils.joinHtml([this.indicators[message_status]), this.messages[message_status]]);
```
This helper would be like interpolateHtml (i.e. it would respect isHtmlSnippet, and escape otherwise), and would return a string flagged as an HTML snippet.